pkg/api/callback.go

```diff
@@ -172,6 +196,7 @@ func (h *RewardHandler) ChannelPointRedeem(w http.ResponseWriter, r *http.Reques
 		zap.L().Info("Submitted song request",
 			zap.String("user", redeemEvent.UserName),
 			zap.String("uri", redeemEvent.UserInput),
+			zap.String("name", sID.String()),
```

Comment:
This isn't a name, right? This is the Spotify track ID

Reply:
You are right haha my bad

```suggestion
			zap.String("spotify_id", sID.String()),
```